server/command.go

```diff
@@ -738,6 +739,25 @@ func (p *Plugin) pipelineRunCommand(ctx context.Context, namespace, ref, channel
 	txt += fmt.Sprintf("**Ref**: %s\n", pipelineInfo.Ref)
 	txt += fmt.Sprintf("**Triggered By**: %s\n", pipelineInfo.User)
 	txt += fmt.Sprintf("**Visit pipeline [here](%s)** \n\n", pipelineInfo.WebURL)
+
+	foundPipelineSubscription := false
+	subs, err := p.GetSubscriptionsByChannel(channelID)
+	if err != nil {
+		p.API.LogWarn("Failed to get subscriptions for the channel", "ChannelID", channelID, "Error", err.Error())
```

Comment:
0/5 we should use the `pluginapi` package when possible

```suggestion
		p.client.Log.Warn("Failed to get subscriptions for the channel", "channel_id", channelID, "error", err.Error())
```

Reply:
@mickmister Updated

Reply:
> 0/5 we should use the pluginapi package when possible

@mickmister Are you referring to this https://pkg.go.dev/github.com/mattermost/mattermost-plugin-api package?